src/scripts/dot-gov.test.js

```diff
@@ -0,0 +1,326 @@
+const {
+  axios,
+  getApp,
+  utils: { cache },
+} = require("../utils/test");
+
+const script = require("./dot-gov");
+
+describe("dot-gov domains", () => {
+  const app = getApp();
+  const mockCache = [
+    {"Agency": "Non-Federal Agency", "City": "Albany", "Domain Name": "ALBANYCA.GOV", "Domain Type": "City", "Organization": "City of Albany", "Security Contact Email": "(blank)", "State": "CA"}, 
+    {"Agency": "Non-Federal Agency", "City": "Belle Plaine", "Domain Name": "BELLEPLAINEIOWA.GOV", "Domain Type": "City", "Organization": "City of Belle Plaine", "Security Contact Email": "(blank)", "State": "IA"},
+    {"Agency": "U.S. Department of Agriculture", "City": "Washington", "Domain Name": "RURAL.GOV", "Domain Type": "Federal - Executive", "Organization": "Rural Development", "Security Contact Email": "cyber.(blank)", "State": "DC"}, 
+    {"Agency": "The Supreme Court", "City": "Washington", "Domain Name": "SUPREMECOURTUS.GOV", "Domain Type": "Federal - Judicial", "Organization": "Supreme Court of the United Statest", "Security Contact Email": "(blank)", "State": "DC"}, 
+    {"Agency": "Government Publishing Office", "City": "Washington", "Domain Name": "USCODE.GOV", "Domain Type": "Federal - Legislative", "Organization": "United States Government Publishing Office", "Security Contact Email": "(blank)", "State": "DC"}, 
+    {"Agency": "Non-Federal Agency", "City": "Arizona City", "Domain Name": "ACSD-AZ.GOV", "Domain Type": "Independent Intrastate", "Organization": "Arizona City Sanitary District ", "Security Contact Email": "(blank)", "State": "AZ"}, 
+    {"Agency": "Non-Federal Agency", "City": "Mechanicsburg", "Domain Name": "EMSCOMPACT.GOV", "Domain Type": "Interstate", "Organization": "Interstate Commission for EMS Personnel Practice", "Security Contact Email": "(blank)", "State": "PA"}, 
+    {"Agency": "Non-Federal Agency", "City": "St Croix", "Domain Name": "VIVOTE.GOV", "Domain Type": "State", "Organization": "Election System of the Virgin Islands", "Security Contact Email": "(blank)", "State": "VI"}, 
+    {"Agency": "Non-Federal Agency", "City": "Ada", "Domain Name": "CHICKASAW-NSN.GOV", "Domain Type": "Tribal", "Organization": "the Chickasaw Nation", "Security Contact Email": "(blank)", "State": "OK"}
+  ];
+  const commonResponse = {
+    icon_emoji: ":dotgov:",
+    thread_ts: "thread id",
+    unfurl_links: false,
+    unfurl_media: false,
+    username: ".Gov"
+  }
+
+  beforeAll(() => {
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  afterAll(() => {
+  });
+
+  it("subscribes to domain requests in two ways", () => {
+    script(app);
+    expect(app.message).toHaveBeenNthCalledWith(
```

Comment:
I think just `toHaveBeenCalledWith` is fine here. As far as the tests go, it seems to me that we don't really care which order these are called in, as long as they're both called.